### PR TITLE
Bugfix/block size

### PR DESCRIPTION
**Pull request by liayoo**
- Include block header & body in measuring block size
- stateProofHash -> state_proof_hash
- Clean up TODOs

### CODE REVIEW

File: blockchain/block.js [hunk 1]
```diff
@@ -66,7 +71,7 @@ class Block {
             number:            ${this.number}
             epoch:             ${this.epoch}
             timestamp:         ${this.timestamp}
-            stateProofHash:    ${this.stateProofHash}
+            state_proof_hash:    ${this.state_proof_hash}
```
Comment (minsulee2) [inline blockchain/block.js:74]: What about 2 spaces back for others!

File: blockchain/block.js [hunk 2]
```diff
@@ -81,6 +86,11 @@ class Block {
     return ChainUtil.hashString(stringify(block.header));
   }
 
+  static getSize(block) {
+    if (!(block instanceof Block)) block = Block.parse(block);
+    return sizeof({...block.header, ...block.body});
```
Comment (minsulee2) [inline blockchain/block.js:91]: Coooooool

**Review by minsulee2 — APPROVED** [review]
LGTM with tiny comments!

File: blockchain/block.js (hunk 1 above)
Comment (liayoo) [reply blockchain/block.js]: Done! Thanks.

**Pull request merged**